ci-operator/config/openshift/openstack-cinder-csi-driver-operator/openshift-openstack-cinder-csi-driver-operator-master.yaml

```diff
@@ -0,0 +1,22 @@
+binary_build_commands: go build cmd/openstack-cinder-csi-driver-operator/main.go
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: gofmt
```

Comment:
use `make verify` and check it includes go fmt

Reply:
ah, you've created a Makefile. Yeah, `make verify` is more convenient in this case.